Week-10/URL_shortener_app/url-shortener-app/src/index.js

```diff
@@ -0,0 +1,20 @@
+import { StrictMode } from "react";
+import { createRoot } from "react-dom/client";
+import App from "./App";
+import Loader from "react-loader-spinner";
```

Comment:
import the npm packages first, leave a line, and then import the local files. makes it more organized.